modules/core/src/main/scala/org/scalasteward/core/io/WorkspaceAlg.scala

```diff
@@ -54,5 +56,16 @@ object WorkspaceAlg {
 
       override def repoDir(repo: Repo): F[File] =
         fileAlg.ensureExists(reposDir / repo.owner / repo.repo)
+
+      override def findSubProjectDirs(repo: Repo): F[List[File]] = {
+        def findSubProject(file: File): List[File] =
+          if (!file.isDirectory) List.empty
+          else if (file.contains(File("build.sbt"))) List(file)
```

Comment:
I tried your commit locally, it does not work as you might think `File("build.sbt")` in WorkspaceAlg is relative to the execution path. Instead it should be something like: `file.path.resolve("build.sbt")`

Reply:
Thanks for testing and suggestion 👍
Updated based on your suggestion.